src/components/settings/General/GeneralReset.vue

```diff
@@ -85,17 +85,18 @@ export default class SettingsGeneralTabResetDatabase extends Mixins(BaseMixin, S
     async loadResetableNamespaces() {
         this.resetableNamespaces = await this.loadBackupableNamespaces()
 
-        if (this.moonrakerComponents.includes('history')) {
-            this.resetableNamespaces.push({
-                value: 'history_jobs',
-                label: this.$t('Settings.GeneralTab.DbHistoryJobs'),
-            })
+        // stop if history is not enabled
+        if (this.moonrakerComponents.includes('history')) return
```

Comment:
shouldn't that be an inverted check?
the comment says `is not enabled` but you check for enabled history

Reply:
Thanks! You're absolutely right,